pkg/storage-init/storage-init.sh

```diff
@@ -6,6 +6,8 @@
 PERSISTDIR=/persist
 CONFIGDIR=/config
 SMART_DETAILS_FILE=/run/SMART_details.json
+POWER_CYCLE_FILE=/run/global/power-cycle
```

Comment:
Why is this in storage-init? @adarsh-zededa

Reply:
Moved back to simpler logic in `storage-init.sh`. 
Instead of initializing and comparing powerCycle count in `storage-init.sh`, we will be writing the current SMART data to a file (like before) and also keep a backup of the previous SMART data. 
All the comparing logic in moved to nodeagent.